### ISSUE TITLE
Unsupported column type found: INT8

### ISSUE THREAD

**Issue by brettgriffin** [fleet-data/fleet-sink-jdbc#7]
Version `0.0.73`

This bug is hit when running a task that performs an `ALTER` on a table, particularly an ALTER that adds a new column of type `bigint`.

```
Unsupported column type found: INT8
java.io.IOException: Unsupported column type found: INT8
	at io.f8k.fleet.sink.redshift.model.RedshiftSchema.getRedshiftType(RedshiftSchema.java:87)
	at io.f8k.fleet.sink.redshift.model.RedshiftSchema.setTypeNamesForColumns(RedshiftSchema.java:92)
	at io.f8k.fleet.sink.redshift.model.RedshiftSchema.build(RedshiftSchema.java:56)
	at io.f8k.fleet.sink.redshift.query.QueryCreateTempTables.createTempTable(QueryCreateTempTables.java:77)
	at io.f8k.fleet.sink.redshift.query.QueryCreateTempTables.getQueryString(QueryCreateTempTables.java:61)
	at io.f8k.fleet.sink.redshift.query.QueryCreateTempTables.buildQuery(QueryCreateTempTables.java:52)
	at io.f8k.fleet.sink.redshift.query.QueryCreateTempTables.<init>(QueryCreateTempTables.java:46)
	at io.f8k.fleet.sink.redshift.service.RedshiftQueryService.createTempTables(RedshiftQueryService.java:34)
	at io.f8k.fleet.common.command.CommandRunner.run(CommandRunner.java:43)
	at io.f8k.fleet.sink.redshift.RedshiftSink.copyFilesToSink(RedshiftSink.java:270)
	at io.f8k.fleet.common.command.CommandRunner.run(CommandRunner.java:43)
	at io.f8k.fleet.sink.redshift.RedshiftSink.run(RedshiftSink.java:57)
	at io.fleet.Runner.runSink(Runner.java:139)
	at io.fleet.Runner.start(Runner.java:75)
	at io.fleet.Runner.main(Runner.java:30)

```

**Comment by jre247** [comment]
@brettgriffin Do you happen to have steps I can follow to recreate this issue?

**Comment by jre247** [comment]
fleetdata/fleet:0.0.72 fixes this issue.

**Comment by jre247** [comment]
Correction: this is still an issue and fleetdata/fleet:0.0.72 did not fix the issue.

**Comment by jre247** [comment]
Fleet 0.0.76 addresses this fix.

**Comment by brettgriffin** [comment]
Confirmed that this is fixed in `0.0.76`
